corehq/apps/hqadmin/_design/filters/doc_types.js

```diff
@@ -0,0 +1,10 @@
+function (doc, req) {
+    var doc_types_string = req.query.doc_types;
+    if (doc_types_string) {
+        var doc_types = doc_types_string.split(' ');
+        if (doc_types.indexOf(doc.doc_type) >= 0) {
+            return true;
+        }
+    }
+    return false;
+}
```

Comment:
Similar to https://github.com/dimagi/fluff/blob/master/fluff/fluff_filter/_design/filters/domain_type.js but this supports multiple doc types